css/homepage.css

```diff
@@ -0,0 +1,158 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  height: auto;
+  font-family: "Inter", sans-serif;
+}
+
+nav {
+  height: 50px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: transparent;
+}
+
+.header-title {
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 28px;
+  color: #28352f;
+  margin-left: 20px;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  gap: 3px;
+  padding-right: 30px;
+  cursor: pointer;
+}
```

Comment:
Dear, we are very close to make it happen 👏 ☺️ . As the previous reviewer suggested, we have to try as much as possible to stick to the Figma design. This will help us produce a professional portifolio with friendly UI. Therefore I suggest you change the color of the humberger to match the Figma design 👍 .

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/219634521-8ee5f6c7-7f7f-484f-90ba-70724e73a41b.png)

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/219634706-de676404-c95f-47ae-9eb8-691c218d6074.png)